azure_pipelines/verify_msalcpp_per_pr_ios.yml

```diff
@@ -48,12 +48,12 @@ jobs:
      arguments: '--skip-checkout --msal-common-repo $(Agent.BuildDirectory)/CommonCore --remote-ref $(commit_number)'
 
   - task: UsePythonVersion@0
-    displayName: 'Switch to use Xcode 15.2'
+    displayName: 'Switch to use Xcode 15.4'
   - script: |
-     /bin/bash -c "sudo xcode-select -s /Applications/Xcode_15.2.app"
+     /bin/bash -c "sudo xcode-select -s /Applications/Xcode_15.4.app"
 
   - template: azure_pipelines/templates/ios-setup.yml@microsoft-authentication-library-for-cpp
-  
+
   - task: PythonScript@0
     name: Build
     displayName: 'Build x64 Debug iOS'
```

Comment:
would this need change in build.py to use the correct ios versions as well ? iOS15 and Simulator 17.5 ?

Reply:
This is calling MSAL-CPP's build.py, so changes to msal-common-for-objc's build.py are out of scope for this PR

Reply:
aah ok, got it